### ISSUE TITLE
Some sites broken by preferences

### ISSUE THREAD

**Issue by crssi** [ghacksuserjs/ghacks-user.js#17]
FYI... @Roman-Nopantski you can delete this post after reading
Nothing new, but there are some sites identified
network.http.sendSecureXSiteReferrer = false > breaks login at aliexpress
dom.indexedDB.enabled = false > breaks Outlook Web Access (at least on Exchange 2013)
full-screen-api.enabled = false > breaks fullscreen video (youtube, etc...)

Cheers

**Comment by Atavic** [comment]
_Better safe than sorry._

**Comment by crssi** [comment]
I do, I find your work here very very good and I am using your user.js as a base, which doesn't mean that I agree with every settings... so I have some differences for sure. ;)
I am just trying to help you guys identifying some stuff out-of-the-box. Nothing more.

**Comment by earthlng** [comment]
@Atavic, I don't mind your "Better safe than sorry" campaign but please don't flip off the few users who want to help out and contribute. @crssi is not asking for help or moaning that we broke his facebook, instead he traced down the problem with specific sites/services to a specific pref, which IMO is valuable information.

@crssi, please keep participating here, I appreciate your input and we can use every help we get. Thank you

**Comment by earthlng** [comment]
- sendSecureXSiteReferrer will be removed soon and we are looking into improving the referer prefs anyway, so I don't think we need to add this to the warning list. Agreed?
- dom.indexedDB.enabled is in the warning list already and has a WARNING
- full-screen-api.enabled - is pretty self-explanatory, no need for a warning IMO, but @Roman-Nopantski you have the last word. If we can agree that we don't need to do anything for this issue, let's close it.

**Comment by Roman-Nopantski** [comment]
Don't get me wrong, people reporting site breakage is a good thing - it will help us build our knowledge for troubleshooting etc. But....

The readme when finished will make it crystal clear that there will be site breakage, and the wiki will provide a troubleshooting guide (how to narrow down a pref as culprit). The readme will explain how to integrate this: read and edit it first, then implement in a new profile or portable and test it for a while etc (a wiki for setting up second profile or grabbing a portable). On top of that we have a troubleshooting section of numbers and warnings (built from 18 months feedback) which will stay in the js so users have a local reference. And lastly we (will) have yet more info/caveats in the section header for 1600 when it is revamped - just like we have warning in section 0400 section header. Not sure what else we can do.

In other words, I expect users to sort their own problems out re site breakage and not have several hundred issues made. We can't cover the top 1 million alexa sites! That said, I do expect people to report issues with major sites.

Agreed, we can close this

**Comment by bonbonboi** [comment]
> In other words, I expect users to sort their own problems out re site breakage and not have several hundred issues made. We can't cover the top 1 million alexa sites! That said, I do expect people to report issues with major sites.

I agree, I don't think someone comes with an issue with some porn sites, lol since most of 'em sucks, but still reporting major, common, famous web sites are welcome. Plus reporting any web site included how the user solved its issue would be nice... I like to read the Issues section, kind of learning curve.
